src/components/DarkModeToggle/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react'
```

Comment:
Noticed that you converted this from TS to JS, maybe it's better to keep it in Typescript?

Reply:
Yeah linter was screaming because of the TS config with gatsby so I converted in dev and didn't bring it back. I was originally the person who made it TS anyway so might as well bring it back